### PR TITLE
Order Samples page and fix home page buttons radius

### PR DESCRIPTION
**Pull request by nikunjsdev**
Order Samples Page and Fix home page button radius

samples

### CODE REVIEW

**Comment by zshanhui** [comment]
Screenshot 2020-05-22 01 26 40

@nikunjsdev Strange styling for price, notice the `$20` for iPhone 6/7/8

File: src/views/Samples/Page.tsx [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+
+      <div className="collection">
+        <div className="collection-wrapper">
+          <FilterCollection />
+        </div>
+        <div className="collection-wrapper">
+          <GridCollection />
+          <ListCollection />
+        </div>
+      </div>
```
Comment (zshanhui) [inline src/views/Samples/Page.tsx]: This whole section should ideally by a component. It's reused on several pages.

File: src/views/Samples/Page.tsx [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+                <div className="wrapper-img-main-inner--img">
+                  {collections[0] ? (
+                    <Link
+                      to={`/collections/cabinets/${
+                        collections[0].id
+                      }/${collections[0].name
+                        .split(regex)
+                        .join("-")
+                        .toLowerCase()}/samples/}`}
+                    >
```
Comment (zshanhui) [inline src/views/Samples/Page.tsx]: You can simplify this by using: `collection[0].slug`

File: src/views/Samples/Page.tsx [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+                    ""
+                  )}
+                </div>
+                <div className="wrapper-img-main-inner--price">
+                  <span className="old-price">
+                    <del>${pricing.priceRange.start.gross.amount}</del>
+                  </span>
+                  <span className="new-price">
+                    ${pricing.priceRange.start.net.amount}
+                  </span>
```
Comment (zshanhui) [inline src/views/Samples/Page.tsx]: Should be a condition here that if `gross.amount` and `net.amount` is the same, it should only show `net.amount`.

File: src/views/Samples/View.tsx [hunk 4]
```diff
@@ -0,0 +1,33 @@
+import { SAMPLES_PER_PAGE } from "@temp/core/config";
+import * as React from "react";
+import { RouteComponentProps } from "react-router-dom";
+import Page from "./Page";
+import { TypedAllSampleQuery } from "./queries";
+// import './scss/index.scss';
+
+type ViewProps = RouteComponentProps<{ id: string }>;
+
+const STATIC_CAT_SAMPLE_ID = "Q2F0ZWdvcnk6MjM=";
```
Comment (zshanhui) [inline src/views/Samples/View.tsx]: Store as TS enum in `src/constants.ts` as this is reused.
https://www.typescriptlang.org/docs/handbook/enums.html
```
export enum SaleorCategoryIds {
  Sample = "Q2F0ZWdvcnk6MjM=",
}
```

Also remember to change in `src/views/CollectionIndex/Samples/View.tsx`

File: src/views/Samples/View.tsx [hunk 5]
```diff
@@ -0,0 +1,33 @@
+import { SAMPLES_PER_PAGE } from "@temp/core/config";
+import * as React from "react";
+import { RouteComponentProps } from "react-router-dom";
+import Page from "./Page";
+import { TypedAllSampleQuery } from "./queries";
+// import './scss/index.scss';
```
Comment (zshanhui) [inline src/views/Samples/View.tsx]: Remove

File: src/views/Samples/queries.ts [hunk 6]
```diff
@@ -0,0 +1,48 @@
...
+      filter: { categories: [$catId] }
+    ) {
+      totalCount
+      edges {
+        node {
+          ...BasicProductFields
+          ...ProductPricingField
+          collections {
+            id
+            name
```
Comment (zshanhui) [inline src/views/Samples/queries.ts]: Get `slug` instead of `name` to use in Link above.

File: src/views/Samples/queries.ts [hunk 7]
```diff
@@ -0,0 +1,48 @@
...
+          }
+          variants {
```
Comment (zshanhui) [inline src/views/Samples/queries.ts]: Do you need `variants`?

File: src/views/Samples/scss/index.scss [hunk 8]
```diff
@@ -0,0 +1,115 @@
...
+.sample-page {
+  padding: 17px;
+  background-color: $alabaster;
+}
+
+.collection {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-bottom: 34px;
```
Comment (zshanhui) [inline src/views/Samples/scss/index.scss]: Use the variable $spacers or `rem` units for padding, margin.

File: src/views/Samples/scss/index.scss [hunk 9]
```diff
@@ -0,0 +1,115 @@
+@import "../../../globalStyles/scss/variables";
+
+.sample-page {
+  padding: 17px;
```
Comment (zshanhui) [inline src/views/Samples/scss/index.scss]: Use the variable $spacers or `rem` units for padding, margin.

File: src/views/Samples/scss/index.scss [hunk 10]
```diff
@@ -0,0 +1,115 @@
...
+        margin-right: 5px;
+
+        & > div {
+          display: flex;
+        }
+      }
+    }
+
+    &__grid {
+      margin-right: 32px;
```
Comment (zshanhui) [inline src/views/Samples/scss/index.scss]: Use the variable $spacers or `rem` units for padding, margin.

File: src/views/Samples/scss/index.scss [hunk 11]
```diff
@@ -0,0 +1,115 @@
...
+    background-color: $white;
+    height: 200px;
+    border-radius: 5px;
+
+    &-inner {
+      border: 1px solid transparent;
+      display: flex;
+      flex-direction: column;
+      height: 100%;
+      padding: 7px 22px;
```
Comment (zshanhui) [inline src/views/Samples/scss/index.scss]: Use the variable $spacers or `rem` units for padding, margin.

**Review by zshanhui — CHANGES_REQUESTED** [review]

**Review by zshanhui — CHANGES_REQUESTED** [review]
@nikunjsdev 
Overall pretty good. Nice job!

There are a few things to fix to make the code more readable. In general, I would try to make more components out of sections of UI groupings. You can also have more than 1 component per file, but only `export` 1 component per file.

File: src/views/Samples/queries.ts (hunk 7 above)
Comment (nikunjsdev) [reply src/views/Samples/queries.ts]: Removed

File: src/views/Samples/queries.ts (hunk 6 above)
Comment (nikunjsdev) [reply src/views/Samples/queries.ts]: Done!

File: src/views/Samples/View.tsx (hunk 5 above)
Comment (nikunjsdev) [reply src/views/Samples/View.tsx]: Removed!

File: src/views/Samples/View.tsx (hunk 4 above)
Comment (nikunjsdev) [reply src/views/Samples/View.tsx]: Done

File: src/views/Samples/Page.tsx (hunk 3 above)
Comment (nikunjsdev) [reply src/views/Samples/Page.tsx]: Done

File: src/views/Samples/Page.tsx (hunk 2 above)
Comment (nikunjsdev) [reply src/views/Samples/Page.tsx]: Done

File: src/views/Samples/scss/index.scss (hunk 8 above)
Comment (nikunjsdev) [reply src/views/Samples/scss/index.scss]: Done

File: src/views/Samples/scss/index.scss (hunk 9 above)
Comment (nikunjsdev) [reply src/views/Samples/scss/index.scss]: Done

File: src/views/Samples/scss/index.scss (hunk 10 above)
Comment (nikunjsdev) [reply src/views/Samples/scss/index.scss]: Done

File: src/views/Samples/scss/index.scss (hunk 11 above)
Comment (nikunjsdev) [reply src/views/Samples/scss/index.scss]: Done

File: src/views/Samples/Page.tsx (hunk 1 above)
Comment (nikunjsdev) [reply src/views/Samples/Page.tsx]: Here, title "Samples" should be in different component and pass it as props from there?

Comment (zshanhui) [reply src/views/Samples/Page.tsx]: Yes, "Samples" should be passed as `title` prop.

File: src/views/Samples/Page.tsx [hunk 12]
```diff
@@ -0,0 +1,73 @@
...
+      <PageHeader
+        back={true}
+        cart={true}
+        search={true}
+        handleClick={handleBack}
+      />
+
+      <div className="browse-cabinet__heading">
+        <h3>Samples</h3>
+      </div>
```
Comment (zshanhui) [inline src/views/Samples/Page.tsx]: This should be inside: `SubHeader`, and expose a prop: `title="Samples"`

It seems like it's the same UI section.

**Review by zshanhui — CHANGES_REQUESTED** [review]
One more change to `SubHeader` Component.

File: src/views/Samples/Page.tsx (hunk 12 above)
Comment (nikunjsdev) [reply src/views/Samples/Page.tsx]: Okay

**Review by zshanhui — APPROVED** [review]
@nikunjsdev  LGTM!

**Pull request merged**